packages/manager/modules/nutanix/src/dashboard/nodes/list/template.html

```diff
@@ -95,7 +99,7 @@
     </oui-datagrid-column>
     <oui-datagrid-column>
         <node-action-menu
-            data-ng-if="$row.possibleActions"
+            data-ng-if="$row.possibleActions && !$row.isResiliated"
```

Comment:
Resiliate doesn't make sense in english
```suggestion
            data-ng-if="$row.possibleActions && !$row.isTerminated"
```